Source/Core/DolphinWX/NetPlay/NetWindow.cpp

```diff
@@ -75,7 +75,13 @@ static std::string BuildGameName(const GameListItem& game)
 	if (game.GetRevision() != 0)
 		return name + " (" + game.GetUniqueID() + ", Revision " + std::to_string((long long)game.GetRevision()) + ")";
 	else
+	{
+		if (game.IsElfOrDol())
```

Comment:
A better way to do this is to check whether the value of `GetUniqueID()` is empty.